tests/pytests/test_config.py

```diff
@@ -565,11 +565,11 @@ def testModuleLoadexNumericParams():
         res = env.cmd('MODULE', 'LIST')
         env.assertEqual(res, [])
         res = env.cmd('MODULE', 'LOADEX', redisearch_module_path,
-                      'CONFIG', configName, minValue,
+                      'CONFIG', configName, configValue,
```

Comment:
We need to update the comment of line 562. 
`Load module using CONFIG and module ARGS, the CONFIG value should take`